src/pages/Home.js

```diff
@@ -0,0 +1,38 @@
+import { Route, Switch } from 'react-router-dom';
+import Calculator from '../components/calculator/Calculator';
+import NavBar from '../components/navBar/NavBar';
+import NotMatch from './NoMatch';
+import Quote from './Quote';
+import './pages.scss';
+
+const Home = () => (
+  <>
+    <NavBar />
+    <Switch>
+      <Route exact path="/">
+        <div className="home">
+          <h2>Welcome</h2>
+          <p>
+            Occaecat quis anim duis occaecat nostrud dolore consequat anim.
+            Elit amet fugiat id nulla mollit.
+            Quis labore fugiat reprehenderit aliquip consectetur deserunt qui amet nostrud.
```

Comment:
- [ ] Kindly avoid lorem ipsum text. It gives the wrong impression to those who look through your repo that you are not detail-oriented.

Reply:
Solved!